### ISSUE TITLE
How to move Y axis to left hand side of chart?

### ISSUE THREAD

**Issue by josiemessa** [wcharczuk/go-chart#38]
By default, when using the series graphs, the Y axis is rendered on the right hand side of the chart, is there a simple way of getting this to be on the left hand side?

**Comment by ssorc** [comment]
Yes, set `AxisType` to `chart.YAxisSecondary` (as defined in axis.go). For example:

	YAxis: chart.YAxis{
            Name: "Elevation",
            AxisType: chart.YAxisSecondary, 
			Style: chart.Style{
				Show: true,
			},
		},

**Comment by apremalal** [comment]
@ssorc  did you get a chance to look at the values being cropped when switching the axis to left side?

**Comment by suifengtec** [comment]
`YAxis: chart.YAxis{
		AxisType: chart.YAxisSecondary, 
},`

**Comment by yuriiSulak** [comment]
YAxis: chart.YAxis{
		AxisType: chart.YAxisSecondary, 

	}, - not work in version 1, and version 2 ; YAxis just gone

**Comment by armanokka** [comment]
+
